### PR TITLE
Add To Calendar Button

### PR DESCRIPTION
**Pull request by CIPHERTron**
- [x] Add Calendar Invite Link for Gaurav in the href before merging
- [x] Add Calendar Invite Link for Juan in the href before merging
- [x] Add Calendar Invite Link for Logan in the href before merging
- [x] Add Calendar Invite Link for Nikita in the href before merging

### CODE REVIEW

File: src/screens/LandingScreen/SpeakerSection.js [hunk 1]
```diff
@@ -179,6 +179,49 @@ const SpeakerSection = () => {
...
+                      color: '#fff'
+                    }}
+                  >
+                    <a
+                      style={{
+                        textDecoration: 'none',
+                        display: 'flex',
+                        color: '#fff'
+                      }}
+                      href="https://google.com/"
```
Comment (abhibhaw) [inline src/screens/LandingScreen/SpeakerSection.js]: Nikita Gandhi Link Here

**Review by abhibhaw — CHANGES_REQUESTED** [review]
@CIPHERTron Contact links need to be changed :(

**Comment by CIPHERTron** [comment]
> @CIPHERTron Contact links need to be changed :(

I've solved this issue in the latest commit @abhibhaw 

The only thing remaining now is to replace the placeholder links with the calendar invite links!
I've attached a preview of the changes below. 

Screenshot (581)

File: src/screens/LandingScreen/SpeakerSection.js [hunk 2]
```diff
@@ -294,6 +336,48 @@ const SpeakerSection = () => {
...
```
Comment (CIPHERTron) [inline src/screens/LandingScreen/SpeakerSection.js]: Juan's Link goes here

File: src/screens/LandingScreen/SpeakerSection.js [hunk 3]
```diff
@@ -422,6 +506,48 @@ const SpeakerSection = () => {
...
```
Comment (CIPHERTron) [inline src/screens/LandingScreen/SpeakerSection.js]: Logan's calendar invite goes here :)

**Comment by abhibhaw** [comment]
Great @CIPHERTron Looks awesome! I am preparing the calendar links then!

**Comment by abhibhaw** [comment]
@CIPHERTron Gaurav Sen: https://calendar.google.com/calendar/u/0/r/eventedit?uid=5f4212d7db36e40015f240cf&location=http://hacknitr.tech/live&ctz=Asia/Calcutta&text=Gaurav%20Sen%20X%20HackNITR%202.0&dates=20210319T180000/20210319T200000&details=This%20is%20the%20reminder%20for%20Gaurav%20Sen%27s%20session%20on%20-%20%22%20System%20Design%20for%20Hackathons%20%22.%20HackNITR%202.0%20will%20be%20conducted%20entirely%20on%20Airmeet%20and%20the%20join%20link%20will%20be%20active%2006%20hrs%20before%20the%20event.%0A%0AJoin%20Link%3A%20http%3A//hacknitr.tech/live

File: src/screens/LandingScreen/SpeakerSection.js (hunk 1 above)
Comment (abhibhaw) [reply src/screens/LandingScreen/SpeakerSection.js]: https://calendar.google.com/calendar/u/0/r/eventedit?uid=5f4212d7db36e40015f240cf&location=http://hacknitr.tech/live&ctz=Asia/Calcutta&text=Nikita%20Gandhi%20X%20HackNITR%202.0&dates=20210312T180000/20210312T200000&details=This%20is%20the%20reminder%20for%20Nikita%20Gandhi%27s%20session%20on%20-%20%22%20Building%20for%20Next%20Billion%20Users%20%22.%20%0AHackNITR%202.0%20will%20be%20conducted%20entirely%20on%20Airmeet%20and%20the%20join%20link%20will%20be%20active%2006%20hrs%20before%20the%20event.%0A%0AJoin%20Link%3A%20http%3A//hacknitr.tech/live

File: src/screens/LandingScreen/SpeakerSection.js (hunk 2 above)
Comment (abhibhaw) [reply src/screens/LandingScreen/SpeakerSection.js]: https://calendar.google.com/calendar/u/0/r/eventedit?uid=5f4212d7db36e40015f240cf&location=http://hacknitr.tech/live&ctz=Asia/Calcutta&text=Juan%20Pablo%20Flores%20X%20HackNITR%202.0&dates=20210321T220000/20210321T230000&details=This%20is%20the%20reminder%20for%20Juan%20Pablo%20Flores%27s%20session.%20HackNITR%202.0%20will%20be%20conducted%20entirely%20on%20Airmeet%20and%20the%20join%20link%20will%20be%20active%2006%20hrs%20before%20the%20event.%0A%0AJoin%20Link%3A%20http%3A//hacknitr.tech/live

File: src/screens/LandingScreen/SpeakerSection.js (hunk 3 above)
Comment (abhibhaw) [reply src/screens/LandingScreen/SpeakerSection.js]: https://calendar.google.com/calendar/u/0/r/eventedit?uid=5f4212d7db36e40015f240cf&location=http://hacknitr.tech/live&ctz=Asia/Calcutta&text=Logan%20Kilpatrick%20X%20HackNITR%202.0&dates=20210320T193000/20210320T210000&details=This%20is%20the%20reminder%20for%20Logan%20Kilpatrick%27s%20session.%20HackNITR%202.0%20will%20be%20conducted%20entirely%20on%20Airmeet%20and%20the%20join%20link%20will%20be%20active%2006%20hrs%20before%20the%20event.%0A%0AJoin%20Link%3A%20http%3A//hacknitr.tech/live

**Pull request merged**